### ISSUE TITLE
App does not start with last used note opened, instead it opens first node in that notebook.

### ISSUE THREAD

**Issue by selivan** [laurent22/joplin#11484]
### Operating system

Windows

### Joplin version

3.1.24

### Desktop version info

Joplin 3.1.24 (prod, win32)

Client ID: 3c3b2cb8e2194f7791b36b2260e2fae2
Sync Version: 3
Profile Version: 47
Keychain Supported: Yes

Revision: d581264

Backup: 1.4.2
Conflict Resolution: 1.2.3
Copy Code Blocks: 1.1.2
Note list and sidebar toggle buttons: 1.0.3

### Current behaviour

1.    Open some notebook with more than 1 note
1.    Select any note other than first in selected sort order
1.    Close Joplin
1.    Open Joplin
1.    Now the first(according to selected sort order) note is opened, not the note that was opened last time

I tried disabling all installed plugins, it didn't help.

### Expected behaviour

After re-opening the application exactly the same note should be selected.

NOTE: opening last used note worked fine while I was using Joplin 2. This started after upgrading to version 3, including latest release.

P.S. I already opened this issue at Jul 14 as #10743, but now github-actions bot closed it because of "prolonged period of inactivity". But even the longest period of inactivity does not make a bug disappear, it just doesn't work like that.

**Comment by pedr** [comment]
I don't think this ever worked, I tested with 2.11.11 (Jun 2023) and it wasn't working.

Looking at the code we only have an activeFolderId and notesParent (which is only an abstraction to normalize the tag/folder/search id):
![image](https://github.com/user-attachments/assets/0ab99ced-274e-4c2f-96bf-28067d90c2b0)

I could try to refactor `activeFolderId` to `activeNoteId`  (I don't think it would make sense to keep track of both), what do you think? @laurent22

**Comment by selivan** [comment]
@pedr That's weird, I remember that after updating to version 3 it became inconvenient to swtich to last note. Bit it seems to be false memory. Maybe I started to often use a note not on top of notebook sort order.

But this would be a really nice feature to implement. Most software (code editors, e-book readers) can remember last opened file and last position in it.

**Comment by yash-writes-code** [comment]
i would like to work on this, could you please point me to specific files where i shall look.
